website/templates/emails/pending_registration_admin.txt.mako

```diff
@@ -6,7 +6,7 @@ You initiated a registration of your project ${project_name}. The proposed regis
 ${initiated_by} has initiated a registration of your project ${project_name}. The proposed registration can be viewed here: ${registration_link}.
 % endif 
 
-If approved, a registration will be created for the project and will be publicly visible immediately.
+If approved, a registration will be created for the project and, if private, will be made public immediately.
```

Comment:
The two "if" clauses make this sentence hard to read. I suggest removing ", if private, ".